iina/DurationDisplayTextField.swift

```diff
@@ -29,7 +29,7 @@ class DurationDisplayTextField: NSTextField {
   }
   
   
-  func updateText(with duration: VideoTime, given current: VideoTime) {
+  func updateText(with duration: VideoTime, given current: VideoTime, speed of: Double) {
```

Comment:
The new parameter label doesn't really fit in with the rest of the function declaration. Could you try rephrasing this?

Reply:
I basically always uses positional parameters, so sorry for the bad taste of label selection
I've changed it to "and speed", is that OK?